docs/setting_up_your_development_environment.md

```diff
@@ -127,13 +144,13 @@ see the PDF, and annotate it with the injected client sidebar.
 
 ### Via HTML
 
-[Via HTML](https://github.com/hypothesis/viahtml) is the component that handles
-HTML pages for Via.  To install and run it:
+[Via HTML](https://github.com/hypothesis/viahtml) is a proxy that injects the Hypothesis client
+into HTML pages so users can annotate them. To install and run it:
 
 ```terminal
 git clone https://github.com/hypothesis/viahtml.git
 cd viahtml
-make devdata
+make services
```

Comment:
Via HTML doesn't have a `make devdata` command and does have a `make services` command. (The cookiecutter will eventually remove between-projects differences like this by giving every project both `make services` and `make devdata` even if they don't do anything in some projects.